website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/setup-azure.md

```diff
@@ -101,16 +100,186 @@ Once you connect your Azure AD app and Azure DevOps, you need to provide dbt Clo
 Your Azure AD app should now be added to your dbt Cloud Account. People on your team who want to develop in dbt Cloud's IDE can now personally [authorize Azure DevOps from their profiles](dbt-cloud/cloud-configuring-dbt-cloud/authenticate-azure).
 
 ## Connecting a service user
-Because Azure DevOps forces all authentication to be linked to a user's permissions, we recommend you create a "service user" to manage access to your dbt Cloud account. For dbt Cloud scheduled runs, you should avoid linking authentication to an individual Azure DevOps user. Primarily this prevents your dbt Cloud production runs from failing if the person leaves your organization causing a loss of read access for the dbt repository. 
+Because Azure DevOps forces all authentication to be linked to a user's permissions, we recommend you create a "service user" in Azure DevOps whose permissions which will be used to power headless actions in dbt Cloud such as deployment runs and CI. A service user is a pseudo user set up in the same way an admin would set up a real user, but it's given permissions specifically scoped for service to service interactions. You should avoid linking authentication to a real Azure DevOps user because if this person leaves your organization, dbt Cloud will lose privileges to the dbt Azure DevOps repositories, causing production runs to fail.
```

Comment:
maybe not the place for this, but I'm pretty sure "service principal" is the Azure-ier name than "service user"
https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/service-accounts-introduction-azure#types-of-azure-active-directory-service-accounts

Reply:
Let's keep as "service user". We're not actually using an Azure "service principal" in the way that Azure describes it. We try to make that explicit here: "A service user is a pseudo user set up in the same way an admin would set up a real user,"